docs/tutorials/holmes-custom-toolsets.rst

```diff
@@ -0,0 +1,156 @@
+Adding Custom Tools to Holmes
+#########################
+
+Holmes allows you to define custom toolsets that enhance its functionality by enabling additional tools to run Kubernetes commands or other tasks.
+
+In this guide, we will show how to add a custom toolset to Holmes in your ``generated_values.yaml`` file.
+
+Updating the Helm Values (``generated_values.yaml``)
+----------------------------------------------------
+
+To add a toolset in Holmes, update the ``generated_values.yaml`` with the following configuration:
+
+.. code-block:: yaml
+
+    enableHolmesGPT: true
+    holmes:
+      additionalEnvVars:
+        - name: ROBUSTA_AI
+          value: "true"
+        - name: OPENAI_API_KEY
```

Comment:
please remove this env var, lines 20-24, because it's not required with Robusta ai anymore